templates/quickstart-bitbucket-dc-with-vpc.template.yaml

```diff
@@ -144,7 +144,7 @@ Metadata:
       InternetFacingLoadBalancer:
         default: Make instance internet facing
       KeyPairName:
-        default: Key Name
+        default: SSH Key-Pair Name (optional)
```

Comment:
/nit `Key Pair` ? https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html

I don't mind any of the versions, I just think we should be consistent.